submarine-cloud-v2/controller.go

```diff
@@ -111,6 +112,34 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 		return fmt.Errorf("failed to wait for caches to sync")
 	}
 
+	// Example: HelmInstall (can be removed in the future):
+	// This is equal to:
+	// 		helm repo add k8s-as-helm https://ameijer.github.io/k8s-as-helm/
+	// .	helm repo update
+	//  	helm install helm-install-example-release k8s-as-helm/svc --set ports[0].protocol=TCP,ports[0].port=80,ports[0].targetPort=9376
+	// Useful Links: 
+	//   (1) https://github.com/PrasadG193/helm-clientgo-example
+	// . (2) https://github.com/ameijer/k8s-as-helm/tree/master/charts/svc
+	klog.Info("[Helm example] Install")
+	helmActionConfig := helm.HelmInstall(
+		"https://ameijer.github.io/k8s-as-helm/",
+		"k8s-as-helm",
+		"svc",
+		"helm-install-example-release",
+		"default",
+		map[string]string {
+			"set": "ports[0].protocol=TCP,ports[0].port=80,ports[0].targetPort=9376",
+		},	
+	)
+
+	klog.Info("[Helm example] Sleep 60 seconds")
+	time.Sleep(time.Duration(60) * time.Second)
```

Comment:
Why does this need to sleep?

Reply:
This is used to observe the behavior of "helm install" and "helm uninstall".
Without sleep, the helm chart will be uninstalled immediately after the helm chart installation, and thus we cannot observe the behavior of "helm install".